### DISCUSSION TITLE
HW 4 Part 1 Question 1

### DISCUSSION THREAD

**Discussion by Sam-Besch** [cooperrc/computational-mechanics#170] [Q&A]
Hey everyone, I am just beginning question 1 of hw 4 and am immediately running into trouble, the spring constants and forces are not defined in the problem so i am not sure how to make the arrays. Anyway lmk if i'm completely wrong or if there's a place to find them

**Comment by DJA17** [comment]
Just define k1 k2 k3 k4 and F as 100 N. Its just easier that way so you can do the problem. Method im pretty sure is the most important when doing homeworks to demonstrate you understand the process no matter what values. Since 100 N makes sense I'd say go with that.

**Comment by cooperrc** [comment]
Sorry! I just hadn't defined F & k values. 

You can use $k_1=k_2=k_3=k_4 = 100$ N/mm and $F=100~N$ 

The springs will all have deflections of $1-4~mm$ then
